### DISCUSSION TITLE
Add Gender Info?

### DISCUSSION THREAD

**Discussion by Yuki-Soul** [Jisco/VisualFamilyTree#23] [Ideas]
I just found this project and interested to use it. i notice that this program is lack of gender (male/female) info. what i expect it would be shown in right-side edit menu to be selected or somewhere in common tab. but i cant find it. is it already added and i the one who cant find it?  i think it is basic info that should be added from start, so is it purposely not added?

**Comment by Jisco** [comment]
Yes, I didn't add the information on purpose. The reason is that I don't know what added value this information should bring. Regardless of gender, multiple marriages or children are possible.

Do you have an example where this information provides any value other than just being "there"? 😃

**Reply by Yuki-Soul** [reply]
Yes, I think the gender should be added there. so our descendant will know who is the mother or father role. 
after all this program is about familytree so our descendant have right to know his ancestor gender instead of wondering it right? people usually look into their ancestor face and compare it to his face. and usually they look it from their same gender ancestors. so i think it is the important information.
wait, are you non-binary person? i guess if that is the problem. then why not add another 3 gender option 
(nonbinary role mother, nonbinary role father, nonbinary role father-mother) then people will be understand it better.

and because the title of the program have word "visual" people expect it easy to be understand with just only see the nodes.
so maybe in every nodes you better adding color on the nodes sides to mark the person gender (like blue or pink as differentiation).
also adding gender icon somewhere in the nodes will make it better. there is icon for non-binary too on the internet. fell free to use it :)

**Comment by Jisco** [comment]
So no, I'm not non-binary or anything like that. I just don't find this information important. Maybe I'll add that in one of the next versions so that you can define it yourself.

If you already want a color scheme, you can already work with the flags. Just invent one for male or female and assign them accordingly. It's basically the same then...

**Reply by Yuki-Soul** [reply]
owh sorry, i write it just in case. 
good to hear that will be added. i will wait for the next version thanks for understanding 👍

**Reply by Jisco** [reply]
No need to apologize for anything. Just wanted to say that's not the reason 😄
